screens/AppNavigator/screens/BalancesScreen/BalancesNavigator.tsx

```diff
@@ -37,6 +43,24 @@ export function BalancesNavigator (): JSX.Element {
           headerBackTitleVisible: false
         }}
       />
+      <BalanceStack.Screen
+        name='TokenDetail'
+        component={TokenDetailScreen}
+        options={({ route }: { route: any }) => {
+          return {
+            headerTitle: () => {
```

Comment:
You need to add `hide back title` to all navigation if you use an alternative header title. Or else it will clip on iOS.

Reply:
resolve in another PR